powersimdata/design/transmission/zones.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
```

Comment:
> I am wondering if the name of the function clearly communicate what is being done. I feel like discarding intra-zone branches should be reflected somehow in the name of the function. What about `calculate_interzone_capacity`?

How about delete the branch filtering line?

Reply:
I guess your second option shall be sum the rateA of all the branches that have at least one end in the zone? Otherwise, it is just be the third, which is my calculation. Please correct me if I misunderstand.

Reply:
I will change the name of method to calculate_interzone_capacity